### PR TITLE
Feature gsp 4041

### PR DESCRIPTION
**Pull request by aimtrettel**

### CODE REVIEW

File: results/test-results.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    <testcase name="Automated tests for the omar-superoverlay API endpoints Should test the header of badKmlQuery" time="0.0630" classname="Should test the header of badKmlQuery">
+    </testcase>
+    <testcase name="Automated tests for the omar-superoverlay API endpoints Should test createKml" time="0.0000" classname="Should test createKml">
+      <failure message="expected 404 to equal 200" type="AssertionError"><![CDATA[AssertionError: expected 404 to equal 200
+    at Context.eval (https://omar-dev.ossim.io/__cypress/tests?p=cypress/integration/APITestingOLD.js:135:38)]]></failure>
+    </testcase>
+    <testcase name="Automated tests for the omar-superoverlay API endpoints Should test the header of createKml" time="0.0790" classname="Should test the header of createKml">
+    </testcase>
+  </testsuite>
+</testsuites>
```
Comment (knaquin) [inline results/test-results.xml:29]: Should this file be here? There are references to omar-superoverlay?

**Review by knaquin — CHANGES_REQUESTED** [review]
There are a few empty png files. What are those about? Should they be there?

File: results/test-results.xml (hunk 1 above)
Comment (aimtrettel) [reply results/test-results.xml:29]: Definitely not, so sorry, there was a mix up when things got committed earlier in our process

**Comment by knaquin** [comment]
We'll have to close this out completely. The radiantcibot password was checked in and will remain in the git history if merged as is. We created a new branch from master and copied over only the final changes to this PR and merged that in.

**Pull request closed without merging**